app/views/homepage/_services_and_information.html.erb

```diff
@@ -0,0 +1,18 @@
+<section class="homepage-services-and-info govuk-width-container" aria-labelledby="services-and-information-label" id="services-and-information">
```

Comment:
What's the value of having that `services-and-information` label here and then a separate `id/labelledby` relationship with another id with a `-label` suffix? What's more, do we need to have this `aria-labelledby` relationship if the heading is the very next element? From a very quick test with VO, it just skips over this `section` element so this might be redundant code.

Reply:
I think that this is probably a hangover from when this was visually hidden - so, as you correctly point out, it's redundant. Removed in https://github.com/alphagov/frontend/pull/3033/commits/f1a0ce477b0a7b49f8ac5a318f0eee8a3f7e6f2c.